tests/Unit/Service/IMipServiceTest.php

```diff
@@ -289,6 +289,7 @@ public function testIsRequest(): void {
 		$mailbox->setAccountId(200);
 		$mailAccount = new MailAccount();
 		$mailAccount->setId(200);
+		$mailAccount->setEmail('vincent@stardew-valley.edu');
```

Comment:
Test failed with `TypeError: Return value of OCA\Mail\Account::getEmail() must be of the type string, null returned`. Reason was the type hint for getEmail. Fix is to use a proper value for the test.